### DISCUSSION TITLE
Display/Modify values in different formats [BCD, HEX, BINARY]

### DISCUSSION THREAD

**Discussion by martinjug** [frangoteam/FUXA#1206] [Q&A]
Hi,

I am working on a project with Siemens Logo where I would like to display/modify weekley timer from PLC in FUXA
Problem is that value that represents time is BCD encoded and you cannot display/modify value in decimal format.
Is there a way to display/modify values from PLC in HEX format instead of decimal?.
Is there any suggestion how to solve this problem?

Example:
In the PLC, Timer off value is set to 23:30 which you can represent in different ways.
2# 0010 0011 0011 0000
16# 2330
10# 1792

![image](https://github.com/frangoteam/FUXA/assets/10573052/07917e35-1eba-4059-b5d8-01273b6ef30b)

Regards,
Martin

**Comment by unocelli** [comment]
Hi, new in master branch the options to modify tag value via script
